packages/shared/src/components/auth/LoginForm.spec.tsx

```diff
@@ -23,7 +23,10 @@ import { AuthContextProvider } from '../../contexts/AuthContext';
 import { formToJson } from '../../lib/form';
 import AuthOptions, { AuthOptionsProps } from './AuthOptions';
 
+let user = null;
+
 beforeEach(() => {
+  nock.cleanAll();
```

Comment:
No waaaayyy. I could have missed this as well in our test suites for `AccountSecurityPage` which could be the reason for the flaky tests/difficulty to do re-work 😢

Reply:
Hmm could actually be!